ledger/complete/wal/checkpointer.go

```diff
@@ -30,6 +32,29 @@ const VersionV1 uint16 = 0x01
 // Version 3 contains a file checksum for detecting corrupted checkpoint files.
 const VersionV3 uint16 = 0x03
 
+// Version 4 contains a footer with node count and trie count (previously in the header).
```

Comment:
We need to flag that this is a breaking change when we want to deploy

Reply:
Great catch. For now, I marked issue #1750 and this PR with "breaking change" label to help us remember.